### PR TITLE
Update get started following Andre's postman collection.

### PR DESCRIPTION
**Pull request by billy-the-fish**

### CODE REVIEW

File: docs/b.get-started.md [hunk 1]
```diff
@@ -1,74 +1,148 @@
+
 # Get started with Monite API
 
-Monite offers the following development solutions to integrate our functionality into your workflow:
+Monite offers the following roles:
```
Comment (YuriKirilin) [inline docs/b.get-started.md:4]: The term "role" is not accurate here. For the roles are what each Entity may assign to its users. 
To classify Monite API users, the backend-team use the term "access mode" or "access layer". See more details here https://gemms.atlassian.net/wiki/spaces/DT/pages/1619525644/Access+tokens
I'd suggest using "access mode".

File: docs/b.get-started.md [hunk 2]
```diff
@@ -1,74 +1,148 @@
...
+
+- Partner - a person who implements Monite in their SASS tool
+- Entity - the customer of a partner
+- End user - the person who uses the software products that access Monite
 
-- White label - get a branded app up and running in 24 hours
-- Embedded - integrate Monite functionality into your app in 2-4 weeks
+All calls to Monite are totally secure, they require an access token to succeed. The information in the token tells Monite the role assigned to the person making the API call. After a Partner contacts Monite, they retrieve access tokens and use the API to set up a space for each entity in Monite.
 
 This section shows you how to create the [authentication tokens](d.authentication.md) and entities that are the first step for all development solutions.
```
Comment (YuriKirilin) [inline docs/b.get-started.md:12]: how to get, not create (it's backend who creates tokens, API consumer gets them and uses them)

File: docs/b.get-started.md [hunk 3]
```diff
@@ -1,74 +1,148 @@
...
-
-    These API calls are available on the entity level: Partner Api - ReDoc. For example, you can start implementing an incoming bills feature via our API.
-
-    POST https://api.dev.monite.dev/partner-api/entities/v1/incoming_bills
-
-
-## Test your app
+    For an authenticated account, Monite returns `200 OK` and you continue this workflow. 
+
+3. **Retrieve a new API token**
```
Comment (YuriKirilin) [inline docs/b.get-started.md:47]: **Retrieve a new partner-level token**

File: docs/b.get-started.md (hunk 1 above)
Comment (billy-the-fish) [reply docs/b.get-started.md]: Done/

File: docs/b.get-started.md (hunk 2 above)
Comment (billy-the-fish) [reply docs/b.get-started.md]: Done

File: docs/b.get-started.md (hunk 3 above)
Comment (billy-the-fish) [reply docs/b.get-started.md]: Done

**Pull request merged**